index.html

```diff
@@ -24,164 +24,165 @@ <h1 id="title">Survey Form</h1>
     <section>
       <div>
         <form id="survey-form" method="POST" action="https://google.com">
-          <div class="rowTab">
-            <div class="labels">
-              <label id="name-label" for="name">* Nome: </label>
-            </div>   
-            <div class="rightTab">
-              <input id="name" type="text" name="nomeUsuario" required placeholder="Digite seu nome">
-            </div> 
-          </div>
-          <div class="rowTab">
-            <div class="labels">
-              <label id="email-label" for="email">* Email: </label>
-            </div>   
-            <div class="rightTab">
-              <input id="email" type="email" name="emailUsuario" required placeholder="Digite seu email">
-            </div> 
-          </div>
-          <div class="rowTab">
-            <div class="labels">
-              <label id="age-label" for="number">* Idade: </label>
```

Comment:
o nome do `for` não está muito descritivo. Sugiro mudar para `age`. Lembra de mudar na label e no input.

Reply:
É que um dos requisitos do Free Code Camp era ter um id="number" dentro do <form>